usagov/usagov-2021
# USAGOV-380-Featured-Pages

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-380

## Description

New template logic for nav pages and left nav

## How To Test
 - Edit a Navigation Cards page (like /about-the-us)
 - Add some pages in the Related Pages field
     - Verify that the added pages are displayed before the normal children
     - Verify that the added pages are displayed in the order that they appear on the edit screen
 - Add one of the existing child pages as a related page
     - Verify that the new page is displayed in the order that it appears on the edit screen
     - Verify that the new page is only shown one time on the nav page
 - Change the page to a Navigation page
     - Verify that the order of all items is the same in the left-nav 
 - Change the page into a Life Events page
     - Verify that the related pages are displayed properly here as well

## Checklist for the Developer



## Checklist for the Peer Reviewers
